Resources/public/templates/dashboard/allcontent-dashboard-block.hbt

```diff
@@ -0,0 +1,36 @@
+<h3 class="ez-allcontent-block-title">All Content</h3>
+<div class="ez-allcontent-block-wrapper ez-asynchronousview">
+    {{#if loadingError}}
+    <p class="ez-asynchronousview-error ez-font-icon">
+        An error occurred while content list.
+        <button class="ez-asynchronousview-retry ez-button ez-font-icon pure-button">Retry</button>
+    </p>
+    {{else}}
+    <table class="pure-table">
+        <thead class="ez-allcontent-block-header">
+            <tr>
+                <th class="ez-allcontent-block-head--title">Title</th>
```

Comment:
we don't use such convention (the `--`) in PlatformUI, for consistency with the rest of the code, please to stick to `-`.